testing/unit/tp/node_account/test_transfer_frozen_to_unfrozen.py

```diff
@@ -0,0 +1,221 @@
+        inputs=INPUTS,
+        outputs=OUTPUTS,
+        dependencies=[],
+        payload_sha512=hash512(data=serialized_transaction_payload),
+        batcher_public_key=RANDOM_NODE_PUBLIC_KEY,
+        nonce=time.time().hex().encode(),
+    )
+
+    serialized_header = transaction_header.SerializeToString()
+
+    transaction_request = TpProcessRequest(
+        header=transaction_header,
+        payload=serialized_transaction_payload,
+        signature=create_signer(private_key=ACCOUNT_FROM_PRIVATE_KEY).sign(serialized_header),
+    )
+
+    mock_context = create_context(account_from_frozen_balance=frozen_value, account_to_unfrozen_balance=UNFROZEN)
+
+    with pytest.raises(InvalidTransaction) as error:
+        NodeAccountHandler().apply(transaction=transaction_request, context=mock_context)
+
+    assert 'Frozen balance is lower than the minimum stake.' == str(error.value)
+
+
+def test_transfer_from_frozen_to_unfrozen():
+    """
+    Case: tokens from reputational frozen balance to reputational unfrozen balance.
+    Expect: frozen and unfrozen balances, stored in state, are changed according to transfer amount.
+    """
+    transfer_value = 10_000
+
+    internal_transfer_payload = NodeAccountInternalTransferPayload()
+    internal_transfer_payload.value = transfer_value
+
+    transaction_payload = TransactionPayload()
+    transaction_payload.method = NodeAccountMethod.TRANSFER_FROM_FROZEN_TO_UNFROZEN
+    transaction_payload.data = internal_transfer_payload.SerializeToString()
+
+    serialized_transaction_payload = transaction_payload.SerializeToString()
+
+    transaction_header = TransactionHeader(
+        signer_public_key=RANDOM_NODE_PUBLIC_KEY,
+        family_name=TRANSACTION_REQUEST_ACCOUNT_HANDLER_PARAMS.get('family_name'),
+        family_version=TRANSACTION_REQUEST_ACCOUNT_HANDLER_PARAMS.get('family_version'),
+        inputs=INPUTS,
+        outputs=OUTPUTS,
+        dependencies=[],
+        payload_sha512=hash512(data=serialized_transaction_payload),
+        batcher_public_key=RANDOM_NODE_PUBLIC_KEY,
+        nonce=time.time().hex().encode(),
+    )
+
+    serialized_header = transaction_header.SerializeToString()
+
+    transaction_request = TpProcessRequest(
+        header=transaction_header,
+        payload=serialized_transaction_payload,
+        signature=create_signer(private_key=ACCOUNT_FROM_PRIVATE_KEY).sign(serialized_header),
+    )
+
+    mock_context = create_context(account_from_frozen_balance=FROZEN, account_to_unfrozen_balance=UNFROZEN)
+
+    NodeAccountHandler().apply(transaction=transaction_request, context=mock_context)
+
+    state_as_list = mock_context.get_state(addresses=[ACCOUNT_ADDRESS_FROM])
+
+    state_as_dict = {}
+    for entry in state_as_list:
+        acc = NodeAccount()
+        acc.ParseFromString(entry.data)
+        state_as_dict[entry.address] = acc
+
+    assert state_as_dict.get(ACCOUNT_ADDRESS_FROM, NodeAccount()).reputation.frozen == FROZEN - transfer_value
```

Comment:
To complicated line. Separate `state_as_dict.get(ACCOUNT_ADDRESS_FROM, NodeAccount())` to the variable.

Reply:
@dmytrostriletskyi, done.